Extension/src/LanguageServer/ui.ts

```diff
@@ -164,9 +164,13 @@ export class UI {
             const isCpp: boolean = (activeEditor.document.uri.scheme === "file" && (activeEditor.document.languageId === "cpp" || activeEditor.document.languageId === "c"));
 
             // It's sometimes desirable to see the config and icons when making settings changes.
-            const isSettingsJson: boolean = ((activeEditor.document.fileName.endsWith("c_cpp_properties.json") || activeEditor.document.fileName.endsWith("settings.json")));
-
-            this.ShowConfiguration = isCpp || isSettingsJson;
+            const isCppPropertiesJson: boolean = activeEditor.document.fileName.endsWith("c_cpp_properties.json");
+            if (isCppPropertiesJson) {
+                vscode.languages.setTextDocumentLanguage(activeEditor.document, "jsonc");
+            }
+            this.ShowConfiguration = isCpp || isCppPropertiesJson ||
+                activeEditor.document.uri.scheme === "output" ||
```

Comment:
This is an unrelated change -- so the configuration doesn't disappear when in C/C++ or Log Diagnostics output.